ENIGMAsystem/SHELL/Widget_Systems/Win32/dialogs.cpp

```diff
@@ -234,6 +235,8 @@ LRESULT CALLBACK InputBoxHookProc(HWND hWndDlg, UINT Msg, WPARAM wParam, LPARAM
     SetDlgItemTextW(hWndDlg, 992, wstrPromptStr);
     SetDlgItemTextW(hWndDlg, 990, wstrTextEntry);
     SetWindowTextW(hWndDlg, wstrWindowStr);
+    if (HideInput == 1)
+      SendDlgItemMessage(hWndDlg, 990, EM_SETPASSWORDCHAR, '*', NULL);
```

Comment:
Why not just toggle the ES_HIDDEN style?
https://github.com/enigma-dev/enigma-dev/blob/ff074e643b6e47cf3b2d73e6260834361c169d75/ENIGMAsystem/SHELL/Widget_Systems/Win32/resources.rc#L17

Reply:
what's the difference?

Reply:
Fair enough then, ok, that's fine.